catwalk/models/metaicl.py

```diff
@@ -16,11 +16,12 @@ def __init__(
         self,
         pretrained_model_name_or_path: str,
         *,
+        likelihood_avging: str = 'tok',
```

Comment:
This isn't the most intuitive abbreviation. I'd recommend just calling it the full `likelihood_averaging`. And use `token` instead of `tok`:
```suggestion
        likelihood_averaging: str = 'token',
```